docs/guides/qedma-qesem.ipynb

```diff
@@ -143,143 +167,171 @@
+    "            None,\n",
+    "            0.03\n",
     "        )\n",
     "    ],\n",
-    "    precision = 0.03,\n",
-    "    run_options = {\n",
-    "        \"backend_name\": \"ibm_brisbane\",\n",
-    "    }\n",
+    "    backend_name = \"ibm_brisbane\",\n",
+    "    options = {\n",
+    "        \"estimate_time_only\": False,\n",
+    "    },\n",
     ")"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "399b0deb",
+   "id": "dfe99043",
    "metadata": {},
    "source": [
-    "<Admonition type=\"note\">\n",
-    "- The `precision` parameter signifies the acceptable error on the expectation values of the observables in absolute value.  Namely, the QPU runtime for mitigation will be determined to provide output values for all the observables of interest that fall within a `1σ` confidence interval of the target precision. If multiple observables are provided, the mitigation will run until the target precision is reached for each of the input observables.\n",
+    "You can use the familiar Qiskit Serverless APIs to check your Qiskit Function workload's status or return results:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "856fe992",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(job.status())\n",
+    "result = job.result()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8b901258",
+   "metadata": {},
+   "source": [
+    "## Function parameters\n",
+    "\n",
+    "\n",
+    "| Name |  Type | Description | Required | Default |  Example |\n",
+    "| -----| ------| ------------| -------- | ------- | -------- |\n",
+    "| `pubs` | [EstimatorPubLike](https://github.com/Qiskit/qiskit/blob/5121a0fda8d4e49d47d16a28e1d21046a09d008b/qiskit/primitives/containers/estimator_pub.py#L202-L207) |This is the main input. The `Pub` contains 2-4 elements: a circuit, one or more observables, 0 or a single set of parameter values, and an optional precision. If a precision was not specified, then the `default_precision` from `options` will be used|  Yes|  N/A |  `[(circuit, [obs1,obs2,obs3], parameter_values, 0.03)]`  |\n",
+    "| `backend_name`| string|Name of the backend to use |No | QESEM will get least busy device reported by IBM| `\"ibm_fez\"`|\n",
+    "| `instance` | string|  The hub/group/project to use in that format |  No |  N/A | `“hub1/group1/project1”`  |\n",
+    "| `options` | dictionary |Input options. See **Options** section for more details. | No |  See the **Options** section for details.    |  `{ default_precision = 0.03, \"max_execution_time\" = 3600, \"transpilation_level\" = 0, \"estimate_time_only\" = False}`  |\n",
+    "\n",
+    "\n",
+    "\n",
+    "### Options\n",
+    "\n",
+    "\n",
+    "| Option |  Choices | Description | Default |\n",
+    "| -----| -----------| -------- | ------- | \n",
+    "| `estimate_time_only` |  True / False  | Specifies whether to execute the mitigation process or only get QPU time estimation | True|\n",
```

Comment:
Does this mean `estimate_time_only` is `True` by default, and does not run on hardware by default?

Reply:
@pandasa123  That is correct. The reason is that error mitigation inherently comes with an exponential overhead, so it's better for the user to request execution explicitly to avoid mistakes where they might start the process without realizing it could consume several hours of their monthly QPU time quota. Moreover, in our SaaS offering, we have implemented a two-stage process where the user cannot begin execution without first estimating the QPU usage. Consider it as a safety switch which protects the user from making mistakes.

Reply:
@assafb, just checked with the team. `estimate_time_only` should be `False` by default to be consistent across all CFs

Reply:
done